Libraries/LibWeb/WebAudio/ControlMessageQueue.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2025, Trey Shaffer <trey@trsh.dev>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Vector.h>
+#include <LibGC/Function.h>
+#include <LibGC/Ptr.h>
+#include <LibJS/Forward.h>
+#include <LibThreading/Mutex.h>
+#include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/Forward.h>
+
+namespace Web::WebAudio {
+
+// https://webaudio.github.io/web-audio-api/#control-message-queue
+class ControlMessageQueue : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(ControlMessageQueue, Bindings::PlatformObject);
+    GC_DECLARE_ALLOCATOR(ControlMessageQueue);
```

Comment:
`ControlMessageQueue` is not a platform object but rather a helper object that will be used by the control thread to perform calls eventually picked up by the rendering thread. You can still make it GC-allocated though.